src/viking.js

```diff
@@ -1,11 +1,105 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health
+        this.strength = strength
+    };
 
+    attack(){
+        return this.strength
+    };
+
+    receiveDamage(damage){
+        this.health = this.health - damage
+    };
+}
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name,health, strength){
+        super(health, strength);
+        this.name=name;
+    }
+
+
+    receiveDamage(damage) {
+        this.health = this.health - damage
+        if(this.health > 0){
+            return `${this.name} has received ${damage} points of damage`
+        } else { 
+            return `${this.name} has died in act of combat`
+        }
+
+    }
+
+    battleCry(){
+        return "Odin Owns You All!"
+    }
+
+
+
+}
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier{
+    constructor(health, strength){
+        super(health, strength);
+    }
+
+    receiveDamage(damage) {
+        this.health = this.health - damage
+
+        if(this.health > 0){
+            
+            return `A Saxon has received ${damage} points of damage`
+        } else { 
+            return `A Saxon has died in combat`
+        }
```

Comment:
**ES6 Tip**: Since this conditional statement only executes one thing in either scope, you can use a ternary operator here. See suggestion below
```suggestion
        return this.health > 0 ? `A Saxon has received ${damage} point of damage` : `A Saxon has died in combat`;
```

It saves you a few lines of code also 👍🏾